roles/openmrs/meta/main.yml

```diff
@@ -2,6 +2,16 @@
 dependencies:
 - role: tomcat7
   install_tomcat_service: true
```

Comment:
How I named the variable "install_tomcat_service" is probably misleading, but this was meant to distinguish whether Tomcat was to be installed as a service (holding multiple apps) or using Tomcat User (with each app running on its own port). So I'm guessing this should be set to `false`.